java-strings-2/README.MD

```diff
@@ -3,3 +3,4 @@
 - [Java Localization – Formatting Messages](https://www.baeldung.com/java-localization-messages-formatting)
 - [Check If a String Contains a Substring](https://www.baeldung.com/java-string-contains-substring)
 - [Removing Stopwords from a String in Java](https://www.baeldung.com/java-string-remove-stopwords)
+- [Empty and Blank Strings in Java](https://www.baeldung.com/empty-blank-strings-java)
```

Comment:
Recommend not to change this file yet since we don't know the final URL. Your editor will add this after the article is published.

Reply:
Removed the article link by reverting the respective commit.